### DISCUSSION TITLE
Console crashing after every generation

### DISCUSSION THREAD

**Discussion by Wang-Thunder** [hlky/stable-diffusion#239] [Q&A]
Figured id ask here before I open an issue.. Every time I generate a prompt since the last update my console will throw a ZeroDivisionError after the generation and It needs to be restarted. 

Anyone seen this issue? 

```
 File "scripts/webui.py", line 1053, in txt2img
    output_images, seed, info, stats = process_images(
    
  File "scripts/webui.py", line 969, in process_images
    grid = image_grid(output_images, batch_size)
    
  File "scripts/webui.py", line 456, in image_grid
    cols = math.ceil(len(imgs) / rows)
    
ZeroDivisionError: division by zero
```

**Comment by Saffire33** [comment]
I have the exact same on a completely fresh install of this and windows 10. Worked just fine on my last boot a few hours ago.
